.github/release.yml

```diff
@@ -0,0 +1,77 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'tag/version'
+        required: true
+        default: '1.0.0'
+
+      action_verification:
```

Comment:
https://github.com/adobe/aepsdk-core-ios/pull/505/files
Can we also port above changes to skip the 'verification' step only if 'create tag' is skipped?